src/neoxp/Commands/ContractCommand.cs

```diff
@@ -3,7 +3,7 @@
 namespace NeoExpress.Commands
 {
     [Command("contract", Description = "Manage smart contracts")]
-    [Subcommand(typeof(Deploy), typeof(Get), typeof(Hash), typeof(Invoke), typeof(List), typeof(Run), typeof(Storage))]
+    [Subcommand(typeof(Deploy), typeof(Get), typeof(Hash), typeof(Invoke), typeof(List), typeof(Run), typeof(Storage), typeof(Download))]
```

Comment:
nit: let's keep these in alphabetical order

Reply:
will fix